### PR TITLE
[DV] Update simulation terminate argument

### PR DESCRIPTION
**Pull request by towoe**
Add description to usage output.
Add short option '-c'.

### CODE REVIEW

File: dv/verilator/simutil_verilator/cpp/verilator_sim_ctrl.cc [hunk 1]
```diff
@@ -68,12 +68,14 @@ void VerilatorSimCtrl::PrintHelp() const {
...
   }
-  std::cout << "-r|--rominit=VMEMFILE  Initialize the ROM with VMEMFILE\n"
-               "-m|--raminit=VMEMFILE  Initialize the RAM with VMEMFILE\n"
-               "-f|--flashinit=VMEMFILE  Initialize the FLASH with VMEMFILE\n"
-               "-h|--help              Show help\n"
+  std::cout << "-r|--rominit=VMEMFILE       Initialize the ROM with VMEMFILE\n"
+               "-m|--raminit=VMEMFILE       Initialize the RAM with VMEMFILE\n"
+               "-f|--flashinit=VMEMFILE     Initialize the FLASH with VMEMFILE\n"
+               "-c|--term-after-cycles=NUM  Terminate simulation after NUM "
+               "cycles\n"
```
Comment (imphil) [inline dv/verilator/simutil_verilator/cpp/verilator_sim_ctrl.cc]: How about
```
               "-c|--term-after-cycles=N Terminate simulation after N cycles\n"
```

Comment (towoe) [reply dv/verilator/simutil_verilator/cpp/verilator_sim_ctrl.cc]: :)
But now that you mentioned it, I noticed that after using only `N` the trace option was using 81 characters so I removed the `a `. Feels better to not exceed the 80 characters.

Comment (imphil) [reply dv/verilator/simutil_verilator/cpp/verilator_sim_ctrl.cc]: I also removed the double whitespace before "Terminate" :-)

Comment (towoe) [reply dv/verilator/simutil_verilator/cpp/verilator_sim_ctrl.cc]: Really? I thought the double spaces were there intentionally to add some separation.

Not to get too much off topic, but should there be a maximum line width for the generated output? We should tackle the important questions first.

Comment (imphil) [reply dv/verilator/simutil_verilator/cpp/verilator_sim_ctrl.cc]: :smile: 80 characters for generated output sounds about right, with the same motivation as 80 characters for source code: that's what your (old school) UNIX terminal is able to display.

Either way, let's not give it too much thought and merge as-is.

Comment (towoe) [reply dv/verilator/simutil_verilator/cpp/verilator_sim_ctrl.cc]: Need to measure a lot of output then.

**Pull request merged**